### DISCUSSION TITLE
Does the mobile app support SSO?

### DISCUSSION THREAD

**Discussion by jthemphill** [community/community#116807] [Mobile] [answered]
My organization recently required Single Sign-On to access its repositories. Since then, I've been unable to look at my organization's code or PRs on my phone. Tapping a link to a PR will skip my browser and open the mobile app, which will show a blank screen with the words "something has gone wrong". Unlike the same page in my web browser, the GitHub mobile app does not give me an opportunity to sign in using my organization's SSO scheme.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.156.0 (127056114)
Device: iPhone 14 (iOS 17.3.1)
</details>

**Comment by Iron-Ham** [comment, marked as answer]
Hi @jthemphill,

I understand your frustration. Depending on the requirements of your organization, you may or may not be able to SSO on the mobile application. Could you try signing out and then signing back in? During the sign-in process, you may be presented by a screen that looks something like this:

![SSO](https://github.com/community/community/assets/3388381/1bf17916-ae97-4f04-a6bb-e49cfd070e32)

During this process, tap the `Authorize` button to be redirected to your organization's SSO provider. Depending on your organization's configuration you may or may not be able to authorize the mobile application.

Let me know how it works for you, 
Hesham

**Reply by jthemphill** [reply]
Yep, signing out and then back in worked for me!
